sample_size/utils/sample_size_run.py

```diff
@@ -0,0 +1,43 @@
+from sample_size.sample_size_calculator.sample_size_calculator import SampleSizeCalculators
+from sample_size.utils.utils import get_power_analysis_input
+from sample_size.utils.utils import get_sample_size
+from sample_size.utils.utils import get_variable_from_input
+
+if __name__ == "__main__":
+    """
+    Calculate sample size based on user inputs for
+        1. metric type: Boolean, Numeric, or Ratio (case insensitive)
+        2. power analysis parameters: alpha, power, and mde
+        3. variable input:
+            * Boolean: probability
+            * Numeric: mean and variance
+            * Ratio: mean and variance of numerator and denominator and their covariance
+
+    For future use in Hivemind, this is a sample code to calculate sample sizes with given parameters
```

Comment:
probably better to not mention any internal thing since this is in a public place. However, the below implementation can probably be added to README.md for the simple usage overview. something like https://github.com/PyGithub/PyGithub#simple-demo